doc-site/components/nve-alert.md

```diff
@@ -146,32 +140,76 @@ Bruk `closable` for å vise en lukke-knapp på høyre side, som skjuler komponen
 
 </CodeExamplePreview>
 
-### Ikoner
+### Duration
 
-Ikoner legges i sporet `icon`.
+Du kan bruke `duration` for å lukke alerten automatisk. Verdien er gitt i millisekunder.
 
 <CodeExamplePreview arrangeComponentsVertically>
 
 ```html
-<nve-alert open>
-  <nve-icon slot="icon" name="check_circle"></nve-icon>
-  med ikon
-</nve-alert>
-<nve-alert open> uten ikon </nve-alert>
+<nve-alert class="alert-duration" duration="2000"> Denne skal forsvinne etter 2 sekunder. </nve-alert>
+<nve-button class="show-alert">Vis alerten</nve-button>
+<script>
+  // Denne snutten vil åpne alerten når man klikker på knappen.
+  const button = document.querySelector('.show-alert');
+  const alert = document.querySelector('.alert-duration');
+  button.addEventListener('click', () => alert.show());
+</script>
 ```
 
 </CodeExamplePreview>
 
-### Ikoner
+### Toast
 
-Ikoner legges i sporet `icon`.
+Du kan selv bestemme hvordan du vil vise alerten i løsningen din. Når du legger til `nve-alert` i DOM-en, er den alltid der selv om `open` er satt til false. `display: none` gjør at skjermlesere ignorerer komponenten, så det er ingen fare ved å ha den i DOM-en hele tiden.
+
+Ofte ønsker man å skjule slike komponenter i SPA-applikasjoner uansett. Vi tilbyr derfor også `toast`-funksjonalitet som gjør dette for deg. Som standard vises toast øverst i høyre hjørne. For å vise en alert som toast, må du wrappe `nve-alert` i en `div` med klassen `toast-stack`. Denne blir injisert i DOM-en utenfor hovedinnholdet. Når alerten lukkes, forsvinner også `toast-stack` sammen med alle `nve-alert`-komponentene. Vi anbefaler ikke å vise mer enn én alert samtidig. Les mer i seksjonen om universell utforming. Den er ikke justbar. Per i dag støttes det kun en animasjon, men kan sikkert prate med designere hvis det er behov for flere typer animasjoner.
+
+<nve-message-card label="Vær oppmerksom" size="compact" variant="warning">
+<p>I dette eksemplet vises det to alert meldinger samtidig når man klikker på begge knappene. Husk at det ikke er best praksis å vise flere alert meldinger samtidig. Bruk det med omhu, og les mer om det i <a href="#universell-utforming">universel utforming</a> seksjonen.</p>
+</nve-message-card>
 
 <CodeExamplePreview arrangeComponentsVertically>
 
 ```html
-<nve-alert open text="Hello">
-  <div><nve-button /></div>
-</nve-alert>
+<div class="toast-container">
+  <nve-alert class="alert-toast" variant="warning" closable label="Toast" text="Obs! Dette er en toast."> </nve-alert>
+  <nve-alert class="alert-toast" variant="success" closable label="Toast" text="Gratulerer! Dette er en toast">
+  </nve-alert>
+  <nve-button class="show-alert-warning">Vis varsel toast</nve-button>
+  <nve-button class="show-alert-success" variant="neutral">Vis suksess toast</nve-button>
```

Comment:
Er det noen grunn til at denne knappen har en annen variant enn den første knappen?

Reply:
for å skille de  siden de skal vise to forskjellige alert varianter.